packages/primitives/src/svg/index.js

```diff
@@ -94,8 +94,7 @@ export const SVG = forwardRef(
 		const appliedProps = {
 			...props,
 			className:
-				classnames( className, { 'is-pressed': isPressed } ) ||
-				undefined,
+				clsx( className, { 'is-pressed': isPressed } ) || undefined,
```

Comment:
I know it was here before, but do we really need the `|| undefined`?

Reply:
Yeah makes no sense. Honestly I didn't really look into these, I just search-replaced.